docs/explanation/logical-axiomatization.md

```diff
@@ -0,0 +1,101 @@
+
+For example, `Betz cell`, a well known cell type which defines layer V of the primary motor cortex, synapses lower motor neurons or spinal interneuron (cell types that reside outside the brain). Having the axiom `'Betz cell' part_of 'cortical layer V'` is definitively wrong. In this case `has soma location` is used. Because of cases like these that are common in neurons, all neurons in CL should use `has soma location`.
+
+### Avoid redundant axioms 
+
+Do not add axioms that are not required. If a parent class already has the axiom, it should not be added to the child class too. 
+For example:
+- `retinal bipolar neuron` is a child of `bipolar neuron` 
+- `bipolar neuron` has the axiom `'has characteristic' some 'cortical bipolar morphology'`
+- Therefore we do not add `'has characteristic' some 'cortical bipolar morphology'` to `retinal bipolar neuron`
+
+Axioms add lines to the ontology, resulting in larger ontologies that are harder to use. 
+
+### Let the reasoner do the work
+
+Asserted is_a parents do not need to be retained as entries in the 'SubClass of' section of the Description window in Protege if the logical definition for a term results in their inference.
+
+For example, `cerebral cortex GABAergic interneuron` has the following logical axioms:
+```
+Equivalent_To
+  'GABAergic interneuron' and 
+  ('has soma location' some 'cerebral cortex')
+```
+We do not need to assert that it is a `cerebral cortex neuron`, `CNS interneuron`, or `neuron of the forebrain` as the reasoner automatically does that. 
+
+We avoid having asserted subclass axioms as these are redundant lines in the ontology which can result in a larger ontology, making them harder to use.
+
+Good practice to let the reasoner do the work: 
+```
+1) If you have created a logical definition for your term, you should delete the asserted is_a parent by clicking on the X to the right term.
+2) Once you synchronize the Reasoner, you will see the reasoned classification of your new term, including the inferred is_a parent(s).
+3) If the inferred classification does not contain the correct parentage, or doesn't make sense, then you will need to modify the logical definition.
+4) If an existing term contains a logical definition and still shows an asserted is_a parent in the 'SubClass of' section, you may delete that asserted parent, as well. Just make sure to run the Reasoner to check that the asserted parent is now replaced with the correct reasoned parent(s).
```

Comment:
... you may delete that asserted parent. (remove "as well")
or 
move this up to 2).